src/imp/tracer_imp.js

```diff
@@ -236,7 +252,7 @@ export default class Tracer extends opentracing.Tracer {
         this.addOption('log_field_value_hard_limit', { type: 'int',     defaultValue: 1024 });
 
         // Meta Event reporting options
-        this.addOption('meta_event_reporting', { type: 'bool', defaultValue: false });
+        this.addOption('disable_meta_event_reporting', { type: 'bool', defaultValue: true });
```

Comment:
Thanks for changing the flag name. There's still a difference between this and the behavior I think we want.

This flag should start with a default of false. After being configured by the user, the option should be a constant for the remaining life of the tracer. It will only be set to true when the user explicitly wants to avoid meta reporting, and only at startup. The user should not need to set this unless meta reporting is causing problems in developer mode.

A separate field to mean "is meta reporting turned on" should also be initialized with a value of false, call it `meta_reporting_on`. After receiving a response from the satellite, the value of `metaReportingOn` is set to `!options.disable_meta_event_reporting && response.commands[...].devMode`. We only send meta-reports when `metaReportingOn` is true.

IOW, the `disable_meta_event_reporting` flag effectively means to disregard the `devMode` setting that would otherwise turn on meta reporting.

Reply:
I see... note, that means we need to change it in all of the other tracers as well.